ci/azure/linux.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         CONDA_ENV: pandas
         TEST_ARGS: "--skip-slow --skip-network"
 
-      py36_locale:
+      py37_locale:
```

Comment:
In `CONDA_PY` we have `37`, as well as in the yaml file, so I assume the name is wrong.